README.md

```diff
@@ -67,13 +105,21 @@ docker exec -it jquerydb mysql -u root -proot
 
 ### Ports
 
-If you already use port 9412 on your host, you can create a `.env` file in this directory and set the following environment variable with a port number of your own choosing:
```

Comment:
Port 9412 was added by @Krinkle on purpose, to not conflict with locally running nginx which I had issues with.

Reply:
Indeed, I would much prefer not to bind port 80 or 8080 by default, but something more personal like 9412.

The problem with a custom port, however, is that you can't have both HTTP and HTTPS on the same port. And it'd require a ton of refactoring and logic to try to maintain a working web server and theme logic that is able to serve different-but-functionally-the-same HTML over entirely different ports. You can't simply generate protocol-relative URLs to solve it, instead the code would have to be aware of both sets of ports. This is what led me to get rid of it.